LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
```

Comment:
An API doesn't need to be part of the mod. I could use the JSON API to interact with a mod that accepts JSON files.